src/__tests__/canonicalFields.int.test.ts

```diff
@@ -1,14 +1,20 @@
 import request from 'supertest';
 import { app } from '../app';
+import {
+  prepareDatabaseForCurrentWorker,
+  cleanupDatabaseForCurrentWorker,
+} from '../test/utils';
 
 const agent = request.agent(app);
 
 describe('/canonicalFields', () => {
   describe('/', () => {
     it('should return HTTP Status Code 200 OK', async () => {
+      await prepareDatabaseForCurrentWorker();
```

Comment:
This should probably be automatic / handled by a `beforeAll` hook for any `integration.test.ts` file.

I was hoping for a way to be more explicit about which tests require a database connection -- e.g. there might be an integration test that does NOT use the database and so spinning up a schema / migrations would be a total waste of time.

Reply:
While I think this is still true -- I think we can do that as an easy refactor later in the name of enabling the baseline ability to write tests.